src/google/protobuf/io/io_win32_unittest.cc

```diff
@@ -164,8 +164,8 @@ void IoWin32Test::SetUp() {
   test_tmpdir.clear();
   wtest_tmpdir.clear();
   DWORD size = ::GetCurrentDirectoryW(MAX_PATH, working_directory);
-  EXPECT_GT(size, 0);
-  EXPECT_LT(size, MAX_PATH);
+  EXPECT_GT(size, 0U);
+  EXPECT_LT(size, (DWORD)MAX_PATH);
```

Comment:
per google C++ style guide "Use C++-style casts like static_cast<float>(double_value), or brace initialization for conversion of arithmetic types like int64_t y = int64_t{1} << 42. Do not use cast formats like (int)x unless the cast is to void. You may use cast formats like `T(x)` only when `T` is a class type."

Reply:
Thanks, updated